README.md

```diff
@@ -290,11 +291,27 @@ Empty object will be returned, if query doesn't find any data.
 
 #### Expected input metadata
 * **SOQL Query** - Input field where you should type the SOQL query. E.g. `"SELECT ID, Name from Contact where Name like 'John Smi%'"`
-
 ### Upsert Object
 Creates or Updates Selected Object.
 Action creates a single object. 
 
+#### List of Expected Config fields
+* **Object** - Input field where you should choose the object type, which you want to find. E.g. `Account`
+* **Type Of Search** - Dropdown list with two values: `Unique Fields` and `All Fields`.
+* **Lookup by field** - Dropdown list with all fields on the selected object if the *Type Of Search* is `All Fields`. If the *Type Of Search* is `Unique Fields`, the dropdown lists instead all fields on the selected object where `type` is `id` or `unique` is `true`.
+* **Update/fill fields** - Dropdown list with all fields on the selected object that can be updated or fill by creation
```

Comment:
I did this because one of the problem with previous action was - **_Upsert field isn't a dropdown_**, it also make use component more comfortable - we can search and select fields (it possible only in config, we can't implement normal search in metadata). 
For example object **_Contact_** have about 43 fields and it's really uncomfortable to find that what you want (specialty if consider that Ctrl+F in chrome [doesn't work normal](http://i.piccy.info/i9/ddb1a8757bc695add165101c488afd65/1628500768/66739/1438208/23809ksnip_20210809_121709.png) on metadata).

Of course, if you think it's unnecessary, I can change it to the previous version.

Reply:
> I did this because one of the problem with previous action was - Upsert field isn't a dropdown, it also make use component more comfortable - we can search and select fields (it possible only in config, we can't implement normal search in metadata).

There is a filter search for fields within the mapper that is located ![here](https://user-images.githubusercontent.com/5710732/128709157-be82db65-eae7-4bfe-b173-96b182a31f1c.png) however, I agree that it could be more visible. We are in the process of coming up with new designs for the mapping experience. I'll add the idea of being able to toggle fields to that process.

In the mean time, I think it is best to keep this logic out of the components.

Reply:
Ok, removed